lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -152,26 +156,28 @@ def multiple_steps_selection(steps)
         ask_to_select('Select step to continue', steps, &:description)
       end
 
-      def ask_to_confirm(message)
-        print "#{message}, [yN]"
-        answer = @stdin.gets.chomp
-        case answer
-        when 'y'
-          true
-        when 'n'
-          false
-        end
+      def ask_decision(message)
+        print "#{message}, [y(yes), n(no), q(quit)]"
+        answer = @stdin.gets || ''
+        filter_decision(answer.downcase.chomp) || ask_decision(message)
       ensure
         clear_line
       end
 
+      def filter_decision(answer)
+        decision = nil
+        DECISION_MAPPER.each do |options, decision_call|
+          decision = decision_call if options.include?(answer)
+        end
+        decision
+      end
+
       def ask_to_select(message, steps)
-        print "#{message}, (q) for quit"
+        print "#{message}, [n(next), q(quit)]"
         answer = @stdin.gets.chomp
-        case answer
-        when 'q'
+        if %w(n no next).include?(answer)
           :quit
```

Comment:
shouln't here be something else: as I read it, it would just return the symbol, but there is no place that would consume it